### PR TITLE
Minor additions - mainly to support listening to news

### PR DESCRIPTION
**Pull request by bjorneg**
- Added paging (Im a newsbuff and Ekot has too many episodes to be handled without paging)
- Added content type. Content type enables a description of
  each program to be displayed in the left screen area.
- Added selectable date+time prefix of each program (again - to know which episode of eg ekot to listen to

### CODE REVIEW

File: addon.py [hunk 1]
```diff
@@ -1,8 +1,12 @@
...
-import datetime
+from datetime import datetime
+from datetime import timedelta
+from datetime import date
+from distutils.util import strtobool
+import sys
 import itertools
 import requests
-from xbmcswift2 import Plugin
+from xbmcswift2 import Plugin,  xbmcgui
```
Comment (dodoadoodoo) [inline addon.py]: Is the xbmcgui import needed?

File: addon.py [hunk 2]
```diff
@@ -223,29 +171,61 @@ def create_broadcast(episode, program_name, prefer_broadcasts):
...
         url = file['url']
-        date_str = file['publishdateutc']
-        date_object = datetime.datetime.fromtimestamp(float(int(date_str[6:-2]) / 1000)).date()
-        date_strftime = date_object.strftime("%d.%m.%Y")
-        duration = file['duration']
-        info = {'duration': duration, 'date': date_strftime, 'title': name, 'album': program_name,
-                'artist': _('Sveriges_Radio')}
-        item = {'label': name, 'path': url, 'icon': logo, 'is_playable': True, 'info': info}
-        items.append(item)
+        print file
```
Comment (dodoadoodoo) [inline addon.py]: Change to log statement and remove print.

**Review by dodoadoodoo — CHANGES_REQUESTED** [review]
Thanks for this. See some minor comments inline. 

Do you think you are able to separate this into one PR for removing the sports stuff and an additional for the paging and date stuff?

**Comment by bjorneg** [comment]
Current commit is the removal of sports broadcats, league, teams change only.
I will create a separate PR for the other stuff

**Comment by bjorneg** [comment]
Current commit is the removal of sports broadcats, league, teams change only.
I will create a separate PR for the other stuff

**Comment by bjorneg** [comment]
Im obviously not that good at git & github - however, now there is two (relevant) commits:
- removal of sport etc
- addition of paging, etc
OK?

**Comment by dodoadoodoo** [comment]
Thanks. Kinda busy with life, will take a closer look any old day now...

**Pull request merged**